src/components/vote-form.js

```diff
@@ -13,45 +13,42 @@ function VoteForm() {
 
 	const getCandidates = async () => {
 		await axios
-			.get(process.env.API_HOST + '/candidates/', candi)
+			.get(process.env.API_HOST + '/candidates/', candidates)
 			.then(({ data }) => {
-				setCandi(data);
+				setCandidates(data);
 			})
-			.catch(function (error) {
+			.catch((error) => {
 				console.log(error);
 			});
 	};
 
-	const voteCandidates = async (person) => {
-		const newUrl =
-			process.env.API_HOST + '/candidates/' + person._id + '/vote/';
+	const voteCandidates = async (candidate) => {
+		const newUrl = `${process.env.API_HOST}/candidates/${candidate._id}/vote/`;
 		await axios
 			.put(newUrl, voteCount)
```

Comment:
```suggestion
			.put(newUrl)
```
request body schema가 명시되지 않은 경우, 안 넘겨주셔도 됩니다!